charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -392,6 +392,10 @@ node:
   # Enable the linux daemonset creation
   enableLinux: true
   enableWindows: false
+  # Warning: This option will be deprecated in a future version of EBS CSI Driver.
```

Comment:
Discussion: Should we be more strong about this? Explicitly stating we will remove it might make users less likely to intend to rely on it long term.

Maybe something like:
```
Warning: This option will be removed in a future release. It is intended as a temporary workaround for users unable to immediately migrate off older kernel versions.
```

Reply:
Agreed. Changed wording in documentation and CLI flag too. (Though we should sync these sources in future)